src/wp-includes/cache.php

```diff
@@ -281,6 +294,25 @@ function wp_cache_flush_runtime() {
 	return wp_cache_flush();
 }
 
+/**
+ * Removes all cache items in a group, if the object cache implementation supports it.
+ * Before calling this method, always check for group flushing support using the
+ * `wp_cache_supports_group_flush()` method.
```

Comment:
```suggestion
 * Removes all cache items in a group, if the object cache implementation supports it.
 *
 * Before calling this method, always check for group flushing support using the
 * `wp_cache_supports_group_flush()` function.
```

:1234: Also applies to the compat version

Reply:
Thanks! I'll set Sergey refine the doc blocks, he does that anyhow.